include/aspect/utilities.h

```diff
@@ -118,6 +118,14 @@ namespace aspect
 
     template <>
     inline
+    unsigned int
+    signaling_nan<unsigned int>()
+    {
+      return std::numeric_limits<unsigned int>::signaling_NaN();
+    }
```

Comment:
break this out into a patch of its own (or simply create a patch that has this hunk and submit it for inclusion). Then rebase your existing branch to get this back onto your branch. This allows to keep separate things separate.